spec/ddtrace/configuration/agent_settings_resolver_spec.rb

```diff
@@ -218,6 +269,8 @@
 
         resolver
       end
+
+      it_behaves_like 'respects user-provided configuration'
```

Comment:
Minor: I don't think we need to test the uds behaviors in these specs, since they are just asserting what happens when multiple configs are provided, and I think it's ok to implicitly assume that if uds doesn't kick in when a single config is provided, it won't kick in when multiple configs are provided.

Reply:
Sounds good!